### ISSUE TITLE
[Bug] Justify doesn't work with 1.8.3 and Firefox 76

### ISSUE THREAD

**Issue by Oremountainflorian** [JohnMcLear/ep_align#19]
## System

OS: Windows as well as iOS and Linux
Browser: Firefox 76
Etherpad: Version 1.8.3

## Problem Description

Align "justify" doesn't work anymore with the new version. This is sad as it's supposed to be the most important align functionality. Still works under Chrome! 

King regards, Flo

**Comment by JohnMcLear** [comment]
Works for me, Firefox 76 on Windows.

Steps to replicate.

git checkout develop (etherpad)
git clone this repo.

**Comment by JohnMcLear** [comment]
@seballot setting ``white-space`` to ``pre-line`` for justify seems to fix this but I wanted to give you the ability to decide what value to set for white-space :)

**Comment by techware01** [comment]
Confirmed: Firefox 76, Safari on MacOS

Works fine in Brave (Chromium Browser)

**Comment by seballot** [comment]
Hi ! sorry for that...

Indeed white-space needed to be fixed, I made a fix in etherpad core (not pushed yet)

I simplified the wrapping thing and use those rules for every browser

```css
  white-space: normal;
  word-wrap: break-word;
  overflow-wrap: break-word;
```

Seems to work nice, but @techware01 could you please test applying those css rules on a safari browser to see if it's works properly?

**Comment by techware01** [comment]
I tried to add it to the element styles of <justify> with the help of the browsers dev tools.
As soon as I paste it the element reloads and resets the styling. But for a short moment the text is displayed correctly. This happens in Safari and Firefox.

**Comment by JohnMcLear** [comment]
Thats expected @techware01 , you will need to modify css files.

I'm really reluctant to see this CSS in core @seballot I'd much rather see it land in the plugin so we reduce the potential damage if this change causes other problems that aren't fully test covered...

**Comment by seballot** [comment]
Hi @JohnMcLear, I understand your point, but I think the change I made re wrapping is a good benefit for etherpad core, because there were those obscure class "doesWrap" "noprewrap", with specific rules, and all those line around wrapping were very old. Now there is only one CSS rule for every browser, and I far I have tested it's working nice. Indeed it could have side effect, but I think we should just spent more time to test than the last time?

**Comment by JohnMcLear** [comment]
Okies but if we push it into core it's gonna take a month or so to fix htis bug because we're not gonna push it out in 1.8.4 (as that's a stability fix release) ;/

**Comment by seballot** [comment]
Why not push it inside 1.8.4? the bug was introduced by me in 1.8.3, so make sense to be included in stability release no?

**Comment by seballot** [comment]
but if you want a direct fix for ep_align, then you can include 
```css
justify {
  white-space: normal;
  word-wrap: break-word;
  overflow-wrap: break-word;
}
``` 
in ep_align

**Comment by JohnMcLear** [comment]
Ah I didn't realize you broken in 1.8.3, okay to fix in 1.8.4 then!

**Comment by techware01** [comment]
Hey updatet etherpad to 1.8.4 and now justify works perfectly fine on firefox and safari! 
Thank you very much for your afford and time you put into etherpad-lite to make it even better :)
